cli/tests/containerize.bats

```diff
@@ -164,33 +150,38 @@ setup() {
   # 一点傻傻地，但是有效。
   mkdir -p "$BATS_TEST_TMPDIR/bin"
   ORIGINAL_PODMAN="$(command -v podman)"
-  cat > "$BATS_TEST_TMPDIR/bin/podman" << EOF
+  if is_local_dev; then
+    # Use the host machine's VM for local development.
+    # The develop is responsible for starting the VM.
+    cat > "$BATS_TEST_TMPDIR/bin/podman" << EOF
+#!/usr/bin/env bash
+HOME=$HOME USER=podman-user XDG_DATA_HOME="$REAL_XDG_DATA_HOME" XDG_CONFIG_HOME="$REAL_XDG_CONFIG_HOME" exec $ORIGINAL_PODMAN "\$@"
+EOF
+  else
+    cat > "$BATS_TEST_TMPDIR/bin/podman" << EOF
 #!/usr/bin/env bash
 HOME=$HOME USER=podman-user exec $ORIGINAL_PODMAN "\$@"
 EOF
+  fi
 
   chmod +x "$BATS_TEST_TMPDIR/bin/podman"
   export PATH="$BATS_TEST_TMPDIR/bin:$PATH"
 }
 
 setup_file() {
+  echo "FLOX_CI_RUNNER: '${FLOX_CI_RUNNER}'" >&3
```

Comment:
I can remove this when I squash if necessary. I was using it for debugging purposes, but we also print a bunch of other paths at startup so maybe this belongs there as well.